sdk/onlineexperimentation/test-resources.bicep

```diff
@@ -0,0 +1,171 @@
+// PARAMETERS
+@description('The location of the resource. By default, this is the same as the resource group.')
```

Comment:
Did you forget to create a tests.yml file to run these tests live?

Reply:
Added a tests.yml